jetson_containers/container.py

```diff
@@ -176,8 +176,9 @@ def build_container(
             # generate the logging file (without the extension)
             log_file = os.path.join(get_log_dir('build'), f"{idx+1:02d}o{len(packages)}_{container_name.replace('/','_')}").replace(':','_')
             jetpack_version = get_jetpack_version()
+            buildkit_val = 0
```

Comment:
I revert this variable from the previous commit, and hardcode it to `0` for now.
When it supports buildkit, we can simply change this line.